chat-staging.html

```diff
@@ -32,30 +32,52 @@
                 submitLabel: "Let's chat!"
             }
         };
+        const defaultLanguage = 'en-US';
+        const helplineLanguage = 'es';
+
         Twilio.FlexWebChat.createWebChat(appConfig).then(webchat => {
             const { manager } = webchat;
-            Twilio.FlexWebChat.MessagingCanvas.defaultProps.predefinedMessage.body = "How can I help?";
-        
-            //Posting question from preengagement form as users first chat message
-             Twilio.FlexWebChat.Actions.on("afterStartEngagement", (payload) => {
-                 const { question } = payload.formData;
-                 if (!question)
-                    return;
+            const serverlessBaseUrl = manager.serviceConfiguration.attributes.serverless_base_url;
+            
+            const twilioStrings = { ...manager.strings }; // save the originals
+            const setNewStrings = newStrings => (manager.strings = { ...manager.strings, ...newStrings });
+            const translationErrorMsg = 'Could not translate, using default';
+
+            const changeLanguageWebChat = async language => {
+              try {
+                const response = await fetch(`${serverlessBaseUrl}/translations/${language}/messages.json`);
```

Comment:
Keep in mind that the chat.html file is simply served from a public Amazon S3 bucket, and is available to anyone.  We do not authenticate the user, and therefore we won't be able to use the Flex JWE to validate function calls.  We can retrieve localized messages via a number of means:
1. Hard-code them straight into this file
2. Store them in a public Twilio asset (and/or expose a public Twilio function to retrieve the asset to ensure CORS is supported).
3. Store them in the same or another S3 bucket.

I am happy with any of these options, and perhaps lean a little toward #1 for now, even though it creates some duplication.

I think the near-term outcome we want is that when people use the production chat, when they select a helpline from the dropdown, they get messages in their language.